README.md

```diff
@@ -3,10 +3,9 @@
 [![Build Status][travis-badge]][travis]
 [![Coverage][coverage-badge]][coverage]
 [![npm version][npm version]][npm version]
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/priceline/design-system.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/priceline/design-system/alerts/)
 ![bundlephobia-badge]
 
-https://priceline.github.io/design-system/
```

Comment:
Are we finally killing the next app?

Reply:
not yet, this was duplicated below as `Docs`